client/layout/logged-out.jsx

```diff
@@ -93,6 +95,8 @@ const LayoutLoggedOut = ( {
 
 			masterbar = <OauthClientMasterbar oauth2Client={ oauth2Client } />;
 		}
+	} else if ( config.isEnabled( 'jetpack-cloud' ) ) {
```

Comment:
Instead of `config.isEnabled( 'jetpack-cloud' )` lets have a master bar config that tells us to use JetpackCloud one. 
And not have it specifically tied to JetpackCloud.

Reply:
Do you mean something like:

```js
config.isEnabled( 'masterbar/jetpack-cloud' )
```